low_battery.cpp

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
+
+#include "config.h"
+#include "low_battery.h"
+#include "gpu.h"
+#include "spi.h"
+
+#ifdef LOW_BATTERY_PIN
+
+#define LOW_BATTERY_ICON_TOP_LEFT_X 10
+#define LOW_BATTERY_ICON_TOP_LEFT_Y 10
+#define LOW_BATTERY_ICON_WIDTH 35
+#define LOW_BATTERY_ICON_HEIGHT 20
+#define LOW_BATTERY_FORE_COLOR 65535
+#define LOW_BATTERY_BACK_COLOR 0
+
+bool lowBattery = false;
+uint64_t lowBatteryLastPolled = 0;
+
+// Battery icon from: https://github.com/martinohanlon/grrl-bat-monitor
+uint16_t lowBatteryIcon [LOW_BATTERY_ICON_HEIGHT][LOW_BATTERY_ICON_WIDTH] = {

```

Comment:
These three variables can be marked `static` - that documents these variables won't escape this compilation unit, and it sometimes helps compiler optimize as it can reason the visibility is more local.